frontend/tasks/dist.js

```diff
@@ -38,16 +39,9 @@ gulp.task('dist-thumbnail-images', () => {
 });
 
 gulp.task('dist-build', done =>
-  runSequence('clean', 'build', 'dist-rev-assets', 'dist-thumbnail-images', done));
+  runSequence('dist-rev-assets', 'dist-thumbnail-images', done));
 
 gulp.task('dist-watch', ['watch'], () =>
   gulp.watch(config.DEST_PATH + '**/*', ['dist-rev-assets']));
 
-gulp.task('dist-server', () => connect.server({
```

Comment:
FWIW, I don't think anyone was using this gulp task - and it was non-SSL anyway.